### ISSUE TITLE
Move Apache license from heaptimer.py into licenses directory?

### ISSUE THREAD

**Issue by pllim** [ejeschke/ginga#587]
I noticed the license while working on #583.

**Comment by ejeschke** [comment]
I agree.  The only issue is that it moves the license away from the code, so you lose the contextual information about where exactly in the code this license applies.  That information can be useful to people wanting to use the code.

**Comment by pllim** [comment]
Why not have it in both places? (I'll adjust the title...)
